cmd/generate.go

```diff
@@ -220,19 +224,10 @@ function copy_result_with_retry() {
     echo "Copy failed with empty result file(s) in "$target_dir", test failed!"
   fi
 
-  if [ "$2" == "tpcc-results" ]
+  if [ -n "$collect_cpu_info" ]
   then
-    result_files=$(find "$target_dir" -type f -name "*.txt")
-    for result_file in $result_files
-    do
-      prev_line=$(tail -2 "$result_file")
-      if [[ "$prev_line" != *efc* ]] || [[ $(tail -1 "$result_file" | awk '{if(int($3) > 87){print "pass"}}') != "pass" ]];
```

Comment:
Thanks for cleaning up `.bak` nonsense :) We'll need to update `analyze` to optionally include "failed" runs. I can do that in my PR [1] after we merge yours.

[1] https://github.com/cockroachlabs/cloud-report/pull/119